lib/dominoes_cases.rb

```diff
@@ -0,0 +1,23 @@
+class DominoesCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/['= -]+/, '_')
+  end
+
+  def workload
+    <<-WL.chomp
+input_dominoes = #{input}
+    output_chain = Dominoes.chain(input_dominoes)
+    #{can_chain ? 'assert_correct_chain(input_dominoes, output_chain)' : 'assert_equal nil, output_chain, "There should be no chain for #{input_dominoes}"'}
```

Comment:
Line is uncomfortably long. Any way to shorten?

Note that it is not so long in the generated output, since only one of two alternatives is taken.